src/utils/getEnv.ts

```diff
@@ -65,14 +69,9 @@ export const getEnv = ({
 			}),
 		}
 
-	// TODO: I would like to do something like this
-	// const temp = getFirstElementfromResource(temperature)?.['5700']
-	// however right now there is this problem: temperature is 'readonly' and cannot be assigned to the mutable
-	/**
-	 * First instance selected when object is multiple instance
-	 * @see {@link ../../adr/004-instance-selected-when-multiple-instance.md}
-	 */
-	const temp = temperature?.[0]?.['5700']
+	const temp = firstElementfromInstances(
```

Comment:
`firstElementfromIntances` should return undefined or the first element in the array without mutation because the name does not tell the caller that they will get readonly version from it.

If you want to the readonly version, I would rather assign to readonly variable 

```typescript
const temp: Readonly<Temperature_3303> = firstElementfromInstances(temperature)
```